alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunner.kt

```diff
@@ -431,6 +448,36 @@ object MonitorRunner : JobRunner, CoroutineScope, AbstractLifecycleComponent() {
         return true
     }
 
+    // Aggregation Monitors use the throttle configurations defined in ActionExecutionPolicy, this method evaluates that configuration.
```

Comment:
We need UTs for this method, please create an issue and lets add later.

Reply:
Sure, made a TODO on the method as well for tracking